src/caja-file-management-properties.ui

```diff
@@ -83,6 +83,9 @@
       <row>
         <col id="0" translatable="yes">By Size</col>
       </row>
+      <row>
+        <col id="0" translatable="yes">By Size on Disk</col>
+      </row>
```

Comment:
Thanks for adding this - would be interesting to find out when/why this got removed

Reply:
Maybe it was just an oversight, wasn't included in https://github.com/mate-desktop/caja/pull/757...

Reply:
@monsta It can be added by providing an entry here.

https://github.com/mate-desktop/caja/blob/9ea3b711a6a24d4b68231711129982a34938479b/src/file-manager/fm-icon-view.c#L1729